src/07_slices.py

```diff
@@ -12,26 +12,26 @@
 a = [2, 4, 1, 7, 9, 6]
 
 # Output the second element: 4:
-print()
+print(a[1:2])
 
 # Output the second-to-last element: 9
-print()
+print(a[len(a)-2])
 
 # Output the last three elements in the array: [7, 9, 6]
-print()
+print(a[3:6])
 
 # Output the two middle elements in the array: [1, 7]
-print()
+print(a[2:4])
 
 # Output every element except the first one: [4, 1, 7, 9, 6]
-print()
+print(a[1:6])
 
 # Output every element except the last one: [2, 4, 1, 7, 9]
-print()
+print(a[0:5])
```

Comment:
```suggestion
print(a[:-1])
```
dont forget you can use negative indexes in python!